routemaster/subscription.rb

```diff
@@ -0,0 +1,22 @@
+module Routemaster
+  class Subscription
+
+    attr_reader :subscriber, :callback, :topics, :events
+
+    def initialize(options)
+      @subscriber = options.fetch('subscriber')
+      @callback   = options.fetch('callback')
+      @topics     = options.fetch('topics')
+      @events     = options.fetch('events').symbolize_keys
+    end
+
+    def attributes
```

Comment:
I'm following the same pattern implemented for `Topic`. I don't have any strong reason to implement it, I see it more as a convenience method (if someone wants to fetch all the attributes as a whole). 
I'm happy to remove it if you think it's not useful 👍